### PR TITLE
62 plans and pricing

### LINKED ISSUE #62: Enterprise support - pricing - JP.U.COM

### PR DESCRIPTION
**Pull request by deadlight**
** REVIEW #66 FIRST **

## Done

Added new plans and pricing page

## QA

- ./run
- Go to http://0.0.0.0:8012/enterprise-support/plans-and-pricing
- Compare with the [copydoc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#)

## Issue / Card

Fixes #62

### CODE REVIEW

**Comment by Greggless** [comment]
Appears to be good, one small change - A text link wraps, could we stick it on a new line, pretty please?

image

File: templates/enterprise-support/plans-and-pricing.html [hunk 1]
```diff
@@ -0,0 +1,1065 @@
...
+                  <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+                  <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <div class="row">
+          <p><small>*最低価格が適用されます</small></p>
+          <a href="/contact-us?product=ubuntu-advantage" class="p-button--positive">Ubuntu Advantageの利用に関するお問い合わせ</a>
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: I believe this button should just be "お問い合わせ"

File: templates/enterprise-support/plans-and-pricing.html [hunk 2]
```diff
@@ -0,0 +1,1065 @@
...
+        <p>お客様のアカウントに対して専任のサポート担当者を依頼できます。</p>
+      </div>
+      <div class="col-4 p-card">
+        <h4>専任のエンジニア</h4>
+        <p>サーバー、クラウド、Kubernetesを対象にお客様のUbuntu運用を専門とするCanonicalのエンジニア。</p>
+      </div>
+    </div>
+    <div class="row" id="fn-fs">
+      <div class="col-12">
+        <p><small><sup>*</sup> サイト全体のUbuntu Advantageのサブスクリプションが必要です</small></p>
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: Missing button under this note.

File: templates/enterprise-support/plans-and-pricing.html [hunk 3]
```diff
@@ -0,0 +1,1065 @@
...
+          <tbody>
+            <tr>
+              <td>ノードあたりの価格（物理） <sup><a href="#fn-ks">*</a></sup></td>
+              <td class="u-align--center">0ドル</td>
+              <td class="p-table__cell--highlight u-align--center">600ドル/年</td>
+              <td class="p-table__cell--highlight u-align--center">1,200ドル/年</td>
+              <td class="p-table__cell--highlight u-align--center">4,380ドル/年</td>
+            </tr>
+            <tr>
+              <td>ノードあたりの価格（物理）* <sup><a href="#fn-ks">*</a></sup>、OpenStack <sup><a href="#fn-ks">**</a></sup></td>
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: This has a "*" and the linked one

File: templates/enterprise-support/plans-and-pricing.html [hunk 4]
```diff
@@ -0,0 +1,1065 @@
...
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">可用性の高い本番稼働レベルのKubernetesをパブリッククラウド、VMware、OpenStack、ベアメタルでデプロイ</li>
+          <li class="p-list__item is-ticked">GPUアクセラレーション</li>
+          <li class="p-list__item is-ticked">永続ボリューム用ストレージ</li>
+          <li class="p-list__item is-ticked">カスタムネットワーキングのオプション</li>
+          <li class="p-list__item is-ticked">管理プラットフォーム</li>
+          <li class="p-list__item is-ticked">プライベートレジストリ</li>
+          <li class="p-list__item is-ticked">ロードバランサー</li>
+          <li class="p-list__item is-ticked">セキュリティ</li>
+          <li class="p-list__item is-ticked">アプリケーションカタログ</li>
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: Missing a bullet point below this one

File: templates/enterprise-support/plans-and-pricing.html [hunk 5]
```diff
@@ -0,0 +1,1065 @@
...
+        </ul>
+      </div>
+    </div>
+    <div class="row">
+      <p><a href="#kubernetes">Kubernetesパッケージに関する詳細はCanonicalにお問い合わせください&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: This is just a paragraph and its missing this link below it.

File: templates/enterprise-support/plans-and-pricing.html [hunk 6]
```diff
@@ -0,0 +1,1065 @@
...
+        <h3><a href="#storage">ストレージ</a></h3>
+        <div class="p-card__footer">
+          <p class="p-heading--four">f最低価格 0.01ドル 1か月、1GBあたり<sup><a href="#fn-pnp-1">*</a></sup>
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row" id="fn-pnp-1">
+      <div class="col-12">
+        <p><small><sup>*</sup> Minimums apply</small></p>
```
Comment (Greggless) [inline templates/enterprise-support/plans-and-pricing.html]:
```suggestion
        <p><small><sup>*</sup> 最低価格が適用されます</small></p>
```

File: templates/enterprise-support/plans-and-pricing.html [hunk 7]
```diff
@@ -0,0 +1,1065 @@
...
+            </tr>
+          </tbody>
+        </table>
+        <p id="fn-os-1">
+          <small><sup>*</sup> 最少12のサポートノード</small>
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <p><a href="/contact-us/product=openstack" class="p-button--positive">お問い合わせ</a> <a href="#consulting">CanonicalのOpenstackコンサルティングパッケージを見る&nbsp;›</a></p>
```
Comment (Greggless) [inline templates/enterprise-support/plans-and-pricing.html]:
```suggestion
      <p><a href="/contact-us/product=openstack" class="p-button--positive">お問い合わせ</a></p>
      <p><a href="#consulting">CanonicalのOpenstackコンサルティングパッケージを見る&nbsp;›</a></p>
```

File: templates/enterprise-support/plans-and-pricing.html [hunk 8]
```diff
@@ -0,0 +1,1070 @@
...
+        </p>
+        <p class="u-no-margin--top" id="fn-oc-2">
+          <small><sup>**</sup> Solidfire、ScaleIOなどをアドオンとして利用できます。追加料金が適用される場合があります。</small>
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-12">
+        <p>OpenStackサービスパッケージに関する詳細 <a href="/contact-us?product=openstack" class="p-button--positive">お問い合わせ</a> <a href="#openstack">CanonicalのOpenstackの価格を見る&nbsp;›</a></p>
```
Comment (Greggless) [inline templates/enterprise-support/plans-and-pricing.html]:
```suggestion
        <p>OpenStackサービスパッケージに関する詳細 <a href="/contact-us?product=openstack" class="p-button--positive">お問い合わせ</a></p>
        <p><a href="#openstack">CanonicalのOpenstackの価格を見る&nbsp;›</a></p>
```

**Comment by kwm14** [comment]
## Design review
Few design updates 😉 it's a consistent change across the board with button styling. Based on copy doc styling of buttons.

### Ubuntu Advantage
- [ ] primary button here instead of a link as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#)

screenshot 2019-01-23 at 17 26 33

### \オプションのアドオン
- [ ] secondary button here instead of primary styling as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#heading=h.c0r1z7ukrvr7)

screenshot 2019-01-23 at 17 30 14

### \オプションのアドオン
- [ ] same as above comment

### OpenStackのサポート
- [ ] secondary button here instead of primary styling as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#heading=h.qw8lg84nd7er)

screenshot 2019-01-23 at 17 33 12

### Kubernetesのサポート
- [ ] secondary button here instead of primary styling as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#heading=h.bmhfbbakobne)

screenshot 2019-01-23 at 17 35 22

### コンサルティングとサービスパッケージ
- [ ] secondary button here instead of primary styling as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#)

screenshot 2019-01-23 at 17 36 57

### CephおよびSwiftストレージ
- [ ] secondary button here instead of primary styling as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#heading=h.zfjr8zvr168y)

screenshot 2019-01-23 at 17 38 56

### Landscapeエンタープライズシステム管理
- [ ] secondary button here instead of primary styling as per [copy doc](https://docs.google.com/document/d/1GG7phQ-7oxotwH2gKCsdAW5lr4KDC7KbWeSUYv-tRg8/edit#heading=h.9q52wbh3nkil)

screenshot 2019-01-23 at 17 40 56

**Comment by kwm14** [comment]
Thanks @deadlight LGTM 👍

File: templates/enterprise-support/plans-and-pricing.html [hunk 9]
```diff
@@ -0,0 +1,1070 @@
...
+{% block title %}プランと価格 | Ubuntu{% endblock %}
+{% block meta_description %}Canonicalのすべてのサポートサービスのまとめです。Ubuntuは、商用にサポートされているLinuxソリューションの中で最高の経済性と最高のコミュニティサポートを提供します。{% endblock %}
+{% block meta_keywords%} OpenStack, クラウド, Ubuntu OpenStack, プライベート クラウド, パブリック クラウド, オープン ソース, IAAS, インフラストラクチャ, サービス, Ubuntu, Ubuntu クラウド インフラストラクチャ,
+Ubuntu クラウド ゲスト, Juju, Maas, オーケストレーション, 展開, プロビジョニング, Jumpstart, Amazon, AWS, Joyent, IBM, HP Cloud, Microsoft Azure, ゲスト,
+OS, Linux, ディストリビューション, OpenStack, Ubuntu OpenStack, Ubuntu Cloud, サーバー, タブレット, スマートフォン, 電話, Windows の代替, Mac の代替, フリー ソフトウェア,
+オープン ソース, シン クライアント, Juju, MAAS, Landscape, Ubuntu Advantage, Ubuntu のダウンロード, 導入, サービス提供, サービス オーケストレーション, サービス, 商用サポート,
+有償サポート, 長期サポート, LTS, 14.04, 15.10, 16.04, Trusty Tahr, Icehouse, Juno, Kilo, Mitaka, Utopic Unicorn, Xenial Xerus {% endblock %}
+
+{% block outer_content %}
+<div id="main-content" class="u-no-margin--top">
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: We can remove the u-no-margin--top here

File: templates/enterprise-support/plans-and-pricing.html [hunk 10]
```diff
@@ -0,0 +1,1070 @@
...
+            </tbody>
+          </table>
+        </div>
+        <a href="/contact-us?product=ubuntu-advantage" class="p-button--neutral">Ubuntu Advantageの利用に関するお問い合わせ</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light is-shallow u-no-padding--bottom">
+    <div class="row" style="overflow-x: auto;">
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]: Why does this row need the inline style?

File: templates/enterprise-support/plans-and-pricing.html [hunk 11]
```diff
@@ -0,0 +1,1070 @@
...
+      <div class="col-12">
+        <h3 id="desktop">デスクトップ向け</h3>
+        <div class="row">
+          <div class="col=10" style="overflow: auto;">
```
Comment (anthonydillon) [inline templates/enterprise-support/plans-and-pricing.html]:
```suggestion
          <div class="col-10" style="overflow: auto;">
```

Comment (deadlight) [reply templates/enterprise-support/plans-and-pricing.html]: Same as above

**Pull request merged**